### PR TITLE
Init migration script for impacts history

### PR DESCRIPTION
**Pull request by Fahmus**
# Description
This PR add migration script to init the history of impact

# Issue
Issue link: BOT-1339

### CODE REVIEW

**Comment by Fahmus** [comment]
jenkins, retest this please

**Comment by Fahmus** [comment]
jenkins, retest this please

**Comment by Fahmus** [comment]
jenkins, retest this please

File: migrations/versions/2b51c35f9879_impact_historization.py [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+    op.execute('DROP TRIGGER IF EXISTS last_disruption_changes on public.disruption')
+    create_disruption_trigger = (
+        'CREATE TRIGGER last_disruption_changes '
+        '   AFTER UPDATE ON public.disruption '
+        '       FOR EACH ROW EXECUTE PROCEDURE log_disruption_update()'
+    )
+    op.execute(create_disruption_trigger)
+
+    op.create_table('impact',
+                    sa.Column('created_at',                 sa.DateTime(),  nullable=False),
```
Comment (dvdn) [inline migrations/versions/2b51c35f9879_impact_historization.py]: Will it be different from 'public_updated_at' ?

Comment (Fahmus) [reply migrations/versions/2b51c35f9879_impact_historization.py]: yes

**Pull request merged**